docs/delegate_call.md

```diff
@@ -1,3 +0,0 @@
-## Delegate call vs call
```

Comment:
Why was this deleted?

Reply:
I'ts not deleted, it's moved to the FAQ, because it makes more sense there.